content/cloud/quickstart.md

```diff
@@ -36,7 +36,13 @@ curl -fsSL https://developer.fermyon.com/downloads/install.sh | bash
 
 This command downloads and unpacks the latest Spin binary in the current directory. You can now run spin using the command `./spin`.
 
-{{ details "Learn more" "[Spin](https://github.com/fermyon/spin) is an open-source project used for creating, developing, building, running, and deploying Spin applications. It is both a CLI tool and a runtime, and provides SDKs for a variety of programming languages, including, but not limited to, Rust, TinyGo, and C#. \n\n The Spin project provides installers that are supported on Linux (amd64), macOS (amd64 and arm64), and Windows(amd64). \n\n The [Spin](/spin) documentation is a good starting place to learn more about using the framework to develop applications."}}
+{{ details "Learn more" "[Spin](https://github.com/fermyon/spin) is an open-source project used for creating, developing, building, running, and deploying Spin applications. It is both a CLI tool and a runtime, and provides SDKs for a variety of programming languages, including, but not limited to, Rust, TinyGo, and C#. \n\n The Spin project provides installers that are supported on Linux (amd64), macOS (amd64 and arm64), and Windows(amd64). It's highly recommended to add Spin to a folder, which is on your path, e.g.:
+<!-- @selectiveCpy -->
+
+```bash
+$ sudo mv spin /usr/local/bin/
+```
```

Comment:
@macolso - the build is failing because you need a blank line here:

> Summary: 1 error(s)
> content/cloud/quickstart.md:44 MD031/blanks-around-fences Fenced code blocks should be surrounded by blank lines [Context: "```"]

You can run `npm run test` locally to see if there are any errors.